srv/start.js

```diff
@@ -18,6 +18,8 @@ var opts = {
   },
   mesh: {
     auto: true,
+    host: envs.WEB_HOST || '127.0.0.1',
+    bases: [envs.BASE_HOST || '127.0.0.1:39999'],
     listen: [
       {pin: 'role:coveralls,cmd:get', model: 'consume'},
```

Comment:
Hey @mihaidma you must add the host also on each pin.

So you will have something like this:

```
{pin: 'role:coveralls,cmd:get', model: 'consume', host: envs.WEB_HOST || '127.0.0.1'}
```

Also this is not WEB_HOST, is COVERALLS_HOST. Each instance has its own host env, do if all services are started on the same machine we can control the variables for each instance.

Only the BASE_HOST is common to all services.